apps/examples/log_dump/log_dump_main.c

```diff
@@ -42,13 +44,20 @@ int log_dump_main(int argc, char *argv[])
 {
 	int ret = 1;
 	char buf[READ_BUFFER_SIZE];
-	int fd = open("/proc/logsave", O_RDONLY);
+	int fd = open("/proc/logsave", O_RDWR);
 
 	if (fd < 0) {
 		printf("Please ensure that procfs is mounted\n");
 		return 0;
 	}
 
+	if (write(fd, LOGDUMP_SAVE_START, strlen(LOGDUMP_SAVE_START) + 1) < 0)
```

Comment:
The log dump starts automatically. The start is actually restart.
Then, don't it correct to be stop and start?

Reply:
Yes, I too agree. According to the current implementation, we get logs for the first time, but we get no logs afterward. So we need to stop log save, print logs, and then start log save again.

Reply:
Change the app : Stop first and start again.
Thanks.